string umrf_graph_name
---
bool success